vnext/src/Libraries/Pressability/Pressability.windows.js

```diff
@@ -639,29 +649,39 @@ export default class Pressability {
     // [Windows
     const keyboardEventHandlers = {
       onKeyUp: (event: KeyEvent): void => {
+        const {onKeyUp} = this._config;
+        onKeyUp && onKeyUp(event);
+
         if (
           event.nativeEvent.code === 'Space' ||
           event.nativeEvent.code === 'Enter' ||
           event.nativeEvent.code === 'GamepadA'
         ) {
-          const {onPressOut, onPress} = this._config;
-
-          // $FlowFixMe: PressEvents don't mesh with keyboarding APIs. Keep legacy behavior of passing KeyEvents instead
-          onPressOut && onPressOut(event);
-          // $FlowFixMe: PressEvents don't mesh with keyboarding APIs. Keep legacy behavior of passing KeyEvents instead
-          onPress && onPress(event);
+          // flowlint sketchy-null:warn, sketchy-null-bool:off
+          if (!event.defaultPrevented) {
```

Comment:
Super minor: can this check just go to the top of the `if` check above?

Reply:
This is currently broken, will fix i follow-up PRs. 
Filed: 
https://github.com/microsoft/react-native-windows/issues/7501
https://github.com/microsoft/react-native-windows/issues/7500